tests/README.md

```diff
@@ -0,0 +1,24 @@
+# Intel® Media SDK Open Source TEst Driver
+
+### This documentation is created to run tests on Linux OS distributions
+
+#### Prerequisites:
+Before running tests ensure that required packages are installed on your system:
+```
+git
+git-lfs
+python >= 3.6
+```
+As for default we assume:
+
+MediaStack tarball installed using install_media.sh or Media SDK has been previously compiled and installed to `/opt/intel/mediasdk`
```

Comment:
```
$ fgrep -rsn install_media.sh
$
```

You refer to non-existing file. Drop it.

> Media SDK has been previously compiled and installed to

Be specific and provide cmake configure instructions. Do you imply where libva should be stored as well?

> As for default we assume:

Any way to change the default?

Reply:
Fixed